From: csagan5 <32685696+csagan5@users.noreply.github.com>
Date: Tue, 16 Oct 2018 05:22:59 +0200
Subject: Disable sending URLs to Google for metrics

Disable the "Make searches and browsing better" feature, which
claims to send URLs of visited pages to Google. If the toggle is not
removed from the UI via a separate patch, it may appear possible to
turn this on, but the change will not be committed.

Taken from the "Disable safe browsing" Bromite patch, which is
originally based on ungoogled-chromium patch.

License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html

Change-Id: I75e381891a09b6621ab5aef79e85022655513bb8
---
 components/unified_consent/unified_consent_service.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/unified_consent/unified_consent_service.cc b/components/unified_consent/unified_consent_service.cc
--- a/components/unified_consent/unified_consent_service.cc
+++ b/components/unified_consent/unified_consent_service.cc
@@ -202,7 +202,7 @@ void UnifiedConsentService::SetUrlKeyedAnonymizedDataCollectionEnabled(
 #endif
 
   pref_service_->SetBoolean(prefs::kUrlKeyedAnonymizedDataCollectionEnabled,
-                            enabled);
+                            false);
 }
 
 void UnifiedConsentService::Shutdown() {
-- 

